examples/inference/benchmark_vllm.py

```diff
@@ -0,0 +1,92 @@
+"""Benchmark the latency of processing a single batch of requests."""
+"""Modified from https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_latency.py
+The only change is printing the throughput.
+"""
+import argparse
+import time
+
+import numpy as np
+import torch
+from tqdm import tqdm
+from vllm import LLM, SamplingParams
+
+
+def main(args: argparse.Namespace):
+    print(args)
+
+    # Process all the requests in a single batch if possible.
+    # NOTE(woosuk): If the request cannot be processed in a single batch,
+    # the engine will automatically process the request in multiple batches.
+    llm = LLM(
+        model=args.model,
+        tokenizer=args.tokenizer,
+        quantization=args.quantization,
+        tensor_parallel_size=args.tensor_parallel_size,
+        max_num_seqs=args.batch_size,
+        max_num_batched_tokens=args.batch_size * args.input_len,
+        trust_remote_code=args.trust_remote_code,
+        dtype=args.dtype,
+    )
+
+    sampling_params = SamplingParams(
+        n=args.n,
+        temperature=0.0 if args.use_beam_search else 1.0,
+        top_p=1.0,
+        use_beam_search=args.use_beam_search,
+        ignore_eos=True,
+        max_tokens=args.output_len,
+    )
+    print(sampling_params)
```

Comment:
remove this or give it a better print format.

Reply:
done